### ISSUE TITLE
Error starting dotCMS when external internet is not available

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#11131]
Recent changes on the **dotCMS/src/main/resources/hibernate.cfg.xml** force the use of internet when starting dotCMS.

## Expected Behavior
The system should be able to start dotCMS offline

## Current Behavior
We changed the dtd used by the **dotCMS/src/main/resources/hibernate.cfg.xml** file, and as the new dtd reference is not inside the hibernate jar dotCMS will try to download the dtd file failing on start if the internet is not available.

```
[24/03/17 11:16:06:073 CST]  INFO cfg.Configuration: configuring from resource: /hibernate.cfg.xml
[24/03/17 11:16:06:073 CST]  INFO cfg.Configuration: Configuration resource: /hibernate.cfg.xml
[24/03/17 11:16:06:210 CST] ERROR cfg.Configuration: problem parsing configuration/hibernate.cfg.xml
com.dotcms.repackage.org.dom4j.DocumentException: hibernate.sourceforge.net Nested exception: hibernate.sourceforge.net
	at com.dotcms.repackage.org.dom4j.io.SAXReader.read(SAXReader.java:484) ~[dot.tika-app-1.3_3.jar:1.3]
	at com.dotcms.repackage.net.sf.hibernate.cfg.Configuration.doConfigure(Configuration.java:967) [dot.hibernate-2.1.7_2.jar:?]
	at com.dotcms.repackage.net.sf.hibernate.cfg.Configuration.configure(Configuration.java:911) [dot.hibernate-2.1.7_2.jar:?]
	at com.dotcms.repackage.net.sf.hibernate.cfg.Configuration.configure(Configuration.java:897) [dot.hibernate-2.1.7_2.jar:?]
	at com.dotmarketing.db.HibernateUtil.buildSessionFactory(HibernateUtil.java:562) [dotcms_4.0_999999.jar:?]
	at com.dotmarketing.db.HibernateUtil.getSession(HibernateUtil.java:656) [dotcms_4.0_999999.jar:?]
	at com.dotmarketing.db.HibernateUtil.startTransaction(HibernateUtil.java:821) [dotcms_4.0_999999.jar:?]
	at com.dotmarketing.startup.StartupTasksExecutor.executeUpgrades(StartupTasksExecutor.java:192) [dotcms_4.0_999999.jar:?]
	at com.liferay.portal.servlet.MainServlet.init(MainServlet.java:117) [dotcms_4.0_999999.jar:?]
	at org.apache.catalina.core.StandardWrapper.initServlet(StandardWrapper.java:1231) [catalina.jar:8.0.18]
	at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:1144) [catalina.jar:8.0.18]
	at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:1031) [catalina.jar:8.0.18]
	at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4901) [catalina.jar:8.0.18]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5188) [catalina.jar:8.0.18]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150) [catalina.jar:8.0.18]
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:725) [catalina.jar:8.0.18]
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:701) [catalina.jar:8.0.18]
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:714) [catalina.jar:8.0.18]
	at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1069) [catalina.jar:8.0.18]
	at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1719) [catalina.jar:8.0.18]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_40]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_40]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_40]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_40]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_40]
```

## Possible Solution
Revert the dtd reference change.

## Your Environment
* dotCMS version used: 4.0.0

**Comment by jgambarios** [comment]
Related to these changes: https://github.com/dotCMS/core/commit/5f3e4ba5356fe18efb548f9e863e7e8c2300cf44#diff-9c74230da3684a37b7834d8bf79771c4L4

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/11132

**Comment by bryanboza** [comment]
Fixed, tested on the distro 4.0.1 // msssql... Now works as expected
